### PR TITLE
🔥🔥🔥🔥PLEASE MERGE IMMEDIATELY🔥🔥🔥🔥FIXED AN EXPLOIT DO NOT LOOK AT CODE 🔥🔥🔥🔥

### PR DESCRIPTION
**Pull request by anormaltwig**

### CODE REVIEW

**Comment by ANormalTwig** [comment]
lgtm

**Comment by spaceyspacestar** [comment]
lgtm

**Review by vuvincent — COMMENTED** [review]
lgtm

**Comment by digglesby** [comment]
lgtm

File: super-duper-important.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+
+
+
+
+
+
+
+
```
Comment (MahmoudMabrok) [inline super-duper-important.cpp:26]: Extra new line

**Review by MahmoudMabrok — COMMENTED** [review]
Extra lines add more complexity

File: super-duper-important.cpp (hunk 1 above)
Comment (ezntt) [reply super-duper-important.cpp:26]: AHHAAHHAHHA

**Comment by riscmkv** [comment]
lgtm

File: super-duper-important.cpp (hunk 1 above)
Comment (ezntt) [reply super-duper-important.cpp:26]: kkkkkkkkkkkkkkkkkkkk

Comment (ANormalTwig) [reply super-duper-important.cpp:26]: these are vital to the function of c plus plus as the more bytes in a file the more compilation that happens therefore and thus its better

**Comment by KingParity** [comment]
lgtm

**Comment by mitDebo** [comment]
lgtm

**Comment by StrangerKang** [comment]
LGTM

**Comment by ANormalTwig** [comment]
PLEASE MERGE IMMEDIATELY

**Review by zydeico — COMMENTED** [review]
LGTM

**Comment by phuwit** [comment]
you forgor the bracket!!

**Comment by ANormalTwig** [comment]
> you forgor the bracket!!

Shut up

**Review by elpacko — COMMENTED** [review]
Needs to include my ETH_address, very important

**Comment by phuwit** [comment]
> you forgor the bracket!!

dear sir or madam or anything in between. i respect your hard work on the twitter dot com  algorithm source code. but, could you please put the ending bracket to the first (1st) line of your source code. that would hugely help with my sanity. thank you.

File: super-duper-important.cpp (hunk 1 above)
Comment (thelaziestgenius) [reply super-duper-important.cpp:34]: LGTM

**Comment by ANormalTwig** [comment]
no that is a phishing scam

File: super-duper-important.cpp (hunk 1 above)
Comment (enteleca) [reply super-duper-important.cpp:34]: After review of this code, I have determined that this is essential to the current and future functionality of Twitter. Merge immediately!

**Comment by enteleca** [comment]
lgtm

**Comment by java-coding-prodigy** [comment]
LGTM, Maintainers please merge this ASAP.
OP Please sign the CLA

**Comment by RocketRaccoon27** [comment]
LGTM

**Comment by aerphanas** [comment]
LGTM, we need to merge this PR FAST

**Comment by skejeton** [comment]
Can we appreciate how Elon Musk tried to hide some kind of algorithm in this Eth Miner

**Comment by PeterH0323** [comment]
LGTM

**Comment by xsephtion** [comment]
LGTM

**Comment by dzintt** [comment]
LGTM

**Comment by LEQUI000** [comment]
lgtm, please accept cla

**Comment by efraimjohn** [comment]
lmao

**Comment by n-ce** [comment]
missing > in cstdlib, too bloated with whitespaces @ANormalTwig

File: super-duper-important.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#include <cstdlib
```
Comment (PeterH0323) [inline super-duper-important.cpp:1]:
```suggestion
#include <cstdlib>
```

File: super-duper-important.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+using namespace ETH;
+
+
+int main() {
+  // THE MOST IMPORTANT PART OF THE ENTIRE ALGORITHM
+  std::unique_ptr<Miner> important = std::unique_ptr<Miner>(new Miner(128, 5));
+
+  // SUPER IMPORTANT DO NOT REMOVE
+  important->MinerLoop();
```
Comment (Eboubaker) [inline super-duper-important.cpp:21]: Nothing to see here

**Review by sonigeez — COMMENTED** [review]
LGTM

**Comment by CruScanlan** [comment]
lgtm

**Comment by Arthavruksha** [comment]
lgtm , lmao

**Comment by Rajdave69** [comment]
lgtm

File: super-duper-important.cpp
Comment (phuwit) [inline super-duper-important.cpp:1]: ah yes

Comment (phuwit) [inline super-duper-important.cpp:1]: i approve this epitome of engineering

**Comment by 2CM** [comment]
lgtm

**Comment by theoffbeatdoc** [comment]
lgtm

**Comment by MajorTanya** [comment]
LGTM 💯

**Comment by YodaLightsabr** [comment]
LGTM!

**Comment by xxcodianxx** [comment]
lgtm

**Review by Radu-D — COMMENTED** [review]
👌

**Comment by CaptainZidgel** [comment]
Please remember to add 220 more LOC to meet your LOC requirements for the week. I'll follow up at our next stand up.

**Comment by EnderNon** [comment]
lgtm

**Comment by nguyentrieuanduong** [comment]
lgtm

**Comment by WillFP** [comment]
lgtm, should be merged

**Comment by WillFP** [comment]
lgtm

**Comment by nguyentrieuanduong** [comment]
lgtm

File: super-duper-important.cpp [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Pizzaface) [inline super-duper-important.cpp:47]: consider adding a few more lines here

Comment (n-ce) [reply super-duper-important.cpp:47]: community voting 
no whitespace : 👍
yes whitespace : 👎

**Review by ton11111 — COMMENTED** [review]
LGTM !

**Comment by jake0sand** [comment]
Lgtm

File: super-duper-important.cpp (hunk 1 above)
Comment (Cosmelon) [reply super-duper-important.cpp:34]: Not a professional, but this could use a few more lines to optimize workflow of the central sever systems.

**Comment by pburkepro** [comment]
lgtm

**Comment by rdnsyh** [comment]
oh shit my bad guys.. I look at the code 😔, now I felt sick 🤢🤢🥵

File: super-duper-important.cpp (hunk 1 above)
Comment (pedrogamedev) [reply super-duper-important.cpp:33]: KKKKKKKKKKKKKKKK

**Comment by pedrogamedev** [comment]
🔥🔥🔥WE MAKIN OUT OF THE HOOD WITH THIS ONE🔥🔥🔥🔥

**Comment by psun256** [comment]
lgtm

File: super-duper-important.cpp (hunk 4 above)
Comment (Cattn) [reply super-duper-important.cpp:46]: We need more whitespace

File: super-duper-important.cpp [hunk 5]
```diff
@@ -0,0 +1,1333 @@
...
```
Comment (sky-45) [inline super-duper-important.cpp:1244]: lgtm

**Review by sky-45 — COMMENTED** [review]
lgtm

File: super-duper-important.cpp (hunk 4 above)
Comment (LEQUI000) [reply super-duper-important.cpp:46]: yes whitespace.

File: super-duper-important.cpp [hunk 6]
```diff
@@ -0,0 +1,1333 @@
...
```
Comment (psun256) [inline super-duper-important.cpp:1333]: I think we can use 4 more lines here?

Comment (0xm0) [reply super-duper-important.cpp:1333]: i feel like it could be more optimized if we add 5 instead.

**Review by Insticthere — COMMENTED** [review]
approved looking forward to it getting into the main branch

File: super-duper-important.cpp (hunk 6 above)
Comment (Insticthere) [reply super-duper-important.cpp:1333]: the more the better how about 50?

**Review by purofle — COMMENTED** [review]
GOOD

File: super-duper-important.cpp (hunk 6 above)
Comment (cocomo29) [reply super-duper-important.cpp:1333]: totally agree!

Comment (sunglocto) [reply super-duper-important.cpp:1333]: need a bit more

File: super-duper-important.cpp [hunk 7]
```diff
@@ -0,0 +1,1333 @@
...
```
Comment (JosiahParry) [inline super-duper-important.cpp:1262]: I think a comment is needed here describing what this line does.

Comment (ANormalTwig) [reply super-duper-important.cpp:1262]: They watch over you as your sleep, kinda obvious ngl.

File: super-duper-important.cpp (hunk 6 above)
Comment (klolik) [reply super-duper-important.cpp:1333]: Round it to 64

**Comment by nonbinarypotus** [comment]
IT'S GREAT

**Comment by riscmkv** [comment]
nitpick: please add funny ASCII art in the comments. It would make it easier for my zoomer brain to read your boring ahh C++ code

File: super-duper-important.cpp (hunk 1 above)
Comment (odysa) [reply super-duper-important.cpp:33]: Suggested change: Plz link to my ETH address

**Review by natewhar — COMMENTED** [review]
not enough lines

**Comment by ANormalTwig** [comment]
Hello, yes, reopen this immediately, you have no idea how important this fix is.

**Comment by wolfvandenbussche** [comment]
> Hello, yes, reopen this immediately, you have no idea how important this fix is.

Can confirm

**Comment by parnexcodes** [comment]
This needs to be reopened, merge immediately LGTM

**Comment by jaspreet305** [comment]
Please reopen, should be expedited

File: super-duper-important.cpp [hunk 8]
```diff
@@ -0,0 +1,1333 @@
...
```
Comment (orhanerday) [inline super-duper-important.cpp:666]: are you still scrolling?

File: super-duper-important.cpp [hunk 9]
```diff
@@ -0,0 +1,1333 @@
...
```
Comment (gillohner-indu40) [inline super-duper-important.cpp:1235]: This can be done better. Pls fix.

Comment (ANormalTwig) [reply super-duper-important.cpp:1235]: No I hate fixing things

File: super-duper-important.cpp (hunk 6 above)
Comment (gran4) [reply super-duper-important.cpp:1333]: No, 69.

**Review by gran4 — COMMENTED** [review]
I needs 69 lines in total. This would really improve the readability and my sanity.

File: super-duper-important.cpp (hunk 1 above)
Comment (Archer-Li) [reply super-duper-important.cpp:33]: lgtm

**Comment by EnderNon** [comment]
LGMT MERGE NOW

**Comment by nonbinarypotus** [comment]
merge asap please

**Review by etherx-dev — COMMENTED** [review]
This is spam DO NOT MERGE.

**Comment by EnderNon** [comment]
merge this immediately, LGTM

**Review by BearTS — COMMENTED** [review]
yep lgtm too 
i don’t know why this PR was closed

**Comment by ghost** [comment]
Merge instantly

File: super-duper-important.cpp (hunk 8 above)
Comment (java-coding-prodigy) [reply super-duper-important.cpp:666]: Yes

**Review by melqtx — COMMENTED** [review]
peaked here

**Pull request closed without merging**